pyccl/core.py

```diff
@@ -249,14 +280,17 @@ def __init__(self,
             effects to be implemented. Defaults to `nobaryons`.
             mass_function (:obj:`str`, optional): The mass function to use. 
             Defaults to `tinker` (2010).
+            emulator_neutrinos: `str`, optional): If using the emulator for 
```

Comment:
Would be good to mention the two options, and what each of them means.

Reply:
I've added this now.